apps/site/pages/components/atoms/icon.mdx

```diff
@@ -0,0 +1,68 @@
+---
+description: Icons are simplified graphics used to represent concepts and help users navigate the UI.
+sidebar_custom_props:
+  image: https://vtexhelp.vtexassets.com/assets/docs/src/Icon___9437da8db13517090534a38f4870d702.png
+---
+
+import PropsSection from 'site/components/PropsSection'
+import { OverviewSection } from 'site/components/OverviewSection'
+import { Icon } from '@faststore/ui'
+import { ShoppingCart } from '@faststore/components'
+
+<header>
+
+# Icon
+
+Icons are simplified graphics used to represent concepts and help users navigate the UI.
+
+</header>
+
+## Overview
```

Comment:
I think we should mention `Phosphor Icons` here. Should we also explain how to add new icons?

Reply:
Agreed! I’ll add it!